CODEOWNERS

```diff
@@ -119,6 +119,7 @@ Kconfig*                                  @tejlmand
 /subsys/shell/                            @nordic-krch
 /subsys/spm/                              @ioannisg
 /subsys/nonsecure/                        @ioannisg
+/subsys/tfm/                              @oyvindronningstad @ioannisg @hakonfam
```

Comment:
wouldn't that be confusing? Since it won't really be its own module, it's just supplementing another module that is defined somewhere else.

Reply:
> wouldn't that be confusing?

in my opinion it is more confusing to have tfm code under `subsys`.
Code not used in the Zephyr build system, but instead is used inside the tfm build system as an external project.

> Since it won't really be its own module, it's just supplementing another module that is defined somewhere else.

This is exactly glue code for the TFM module. Not glue code to integrate TFM into Zephyr build system, but glue code to integrate NCS repo into TFM build system. So just in the opposite direction.
And if we want in future, we could have a NCS specific `modules/tfm/CMakeLists.txt` that would be used as glue code, where we could even adjust what variables are fed to TFM build, and that way handle the path much cleaner.
But that would have to wait for: https://github.com/zephyrproject-rtos/zephyr/pull/30905

Reply:
Fair point, I'll move it. But how about the code now in subsys/tfm/CMakeLists.txt and subsys/tfm/Kconfig?

Reply:
Well maybe not. I'll move that as well.

Reply:
Moved to modules/tfm